meta-refkit/conf/distro/include/refkit-development.inc

```diff
@@ -33,3 +33,4 @@ REFKIT_IMAGE_BUILD_MODE_SELECTED = "1"
 
 REFKIT_IMAGE_COMPUTERVISION_EXTRA_FEATURES_append = " computervision-test"
 REFKIT_IMAGE_FEATURES_COMMON_append = " common-test"
+BAD_RECOMMENDATIONS_remove = "bluez5-client"
```

Comment:
When I only add the packagegroup entry, the build fails with a message indicating there is no provider for bluez5-client. I had to add the BAD_RECOMMENDATIONS_remove entry for the build to work. I had left the packagegroup entry to indicate the dependency for bluetoothctl in common-test, but I´ll test again without it.